src/EditorFeatures/Test/Tagging/AsynchronousTaggerTests.cs

```diff
@@ -103,16 +103,15 @@ public void TestSynchronousOutlining()
 
                 var document = workspace.Documents.First();
                 var textBuffer = document.TextBuffer;
-                var tagger = tagProvider.CreateTagger<IOutliningRegionTag>(textBuffer);
+                var tagger = tagProvider.CreateTagger<IOutliningRegionTag>(textBuffer) as IAccurateTagger<IOutliningRegionTag>;
```

Comment:
Can we do a strict cast here, or at least assert it's non-null so tests don't go randomly crashing on null?